massa-protocol-worker-2/src/connectivity.rs

```diff
@@ -121,16 +122,12 @@ pub fn start_connectivity_thread(
                     // Check if we need to connect to peers
                     let nb_connection_to_try = {
                         let active_connections = manager.active_connections.read();
-                        let connection_to_try = active_connections.max_out_connections - active_connections.nb_out_connections;
-                        if connection_to_try <= 0 {
-                            continue;
-                        }
-                       connection_to_try
+                        active_connections.max_out_connections - active_connections.nb_out_connections
```

Comment:
think this can be refactor like that. 

max_out_connections and nb_out_connections are `usize` and the minimal value is 0.

Reply:
i will add it